src/components/shared/footer/status-badge/status-badge.jsx

```diff
@@ -22,9 +22,16 @@ const statusData = {
 };
 
 const fetchStatus = async () => {
-  const res = await fetch('https://neonstatus.com/summary.json');
+  const res = await fetch('https://statuspage.incident.io/neondatabase/api/v1/summary');
```

Comment:
we need to use the same domain. `https://statuspage.incident.io/neondatabase/api/v1/summary` -> `https://neonstatus.com/api/v1/summary`